src/backend/distributed/commands/foreign_data_wrapper.c

```diff
@@ -90,9 +91,14 @@ NameListHasFDWOwnedByDistributedExtension(List *FDWNames)
 		ObjectAddress extensionAddress = { 0 };
 		ObjectAddress FDWAddress = GetObjectAddressByFDWName(strVal(FDWValue), false);
 
-		if (IsObjectAddressOwnedByExtension(&FDWAddress, &extensionAddress))
+		ObjectAddress *copyFDWAddress = palloc0(sizeof(ObjectAddress));
```

Comment:
can we avoid copyFDWAddress and copyExtensionAddress?

Reply:
Only copyExtensionAddress